### PR TITLE
added cloud foundry to deployment docs

### PR DESCRIPTION
**Pull request by eddiewebb**
Adds cloud foundry section to deployment guide, uses blue/green deployment.

I wasn't sure if it was worth noting a full config but I have used this in https://github.com/eddiewebb/demo-blueskygreenbuilds/blob/master/.circleci/config.yml

### CODE REVIEW

**Comment by ryanwohara** [comment]
LGTM

File: jekyll/_cci2/deployment-integrations.md [hunk 1]
```diff
@@ -158,6 +158,97 @@ workflows:
...
+            branches:
+              only: master
+      - live-deploy:
+          requires:
+            - hold
+          filters:
+            branches:
+              only: master
+```
+
```
Comment (smart-alek) [inline jekyll/_cci2/deployment-integrations.md]: Some extra newlines here. On purpose?

File: jekyll/_cci2/deployment-integrations.md [hunk 2]
```diff
@@ -228,7 +319,7 @@ In the following example, if `build-job` passes and the current branch was the m
          - run:
              name: Deploy Master to GKE
              command: ./deploy.sh
-
```
Comment (smart-alek) [inline jekyll/_cci2/deployment-integrations.md]: Nice, thanks!

File: jekyll/_cci2/deployment-integrations.md [hunk 3]
```diff
@@ -158,6 +158,97 @@ workflows:
...
+      cf stop app-name
+      # delete previous version
+      cf delete app-name -f
+      # Switch name of "dark" version to claim correct name
+      cf rename app-name-dark app-name
+```
+
+### Manual Approval
+
+For additional control or validation, you can add a manual "hold" step between the blue & green steps as shown in the sample workflow below.
```
[the remark this answers is not in the document]
Comment (smart-alek) [reply jekyll/_cci2/deployment-integrations.md]: https://github.com/circleci/circleci-docs/wiki/Style-Guide#-so-on

**Pull request merged**